### PR TITLE
Added Date Only Feature 📅

### PR DESCRIPTION
**Pull request by MarketingPip**
## Summary

Option to format time stamp. 

## Closing Issues
Closes #59 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [X] Improvements to existing code, such as refactoring or optimizations (non-breaking)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] I have read the [**CONTRIBUTING**](https://github.com/cicirello/.github/blob/main/CONTRIBUTING.md) document.
- [X] My code follows the code style of this project.
- [X] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

---

I think this is good to go! :+1: Feel free to add any tests and if I am lucky any references to tutorials so I can learn for future on my own. 

ps; did not mean to delete other PR history :disappointed: - lesson learned tho...

### CODE REVIEW

**Comment by MarketingPip** [comment]
Note; I didn't realize I pushed your older Dockerfile :disappointed: I can re-submit again if needed to keep commit history down...

File: generatesitemap.py [hunk 1]
```diff
@@ -220,8 +220,12 @@ def lastmod(f) :
     mod = subprocess.run(['git', 'log', '-1', '--format=%cI', f],
                     stdout=subprocess.PIPE,
                     universal_newlines=True).stdout.strip()
+    print(date_only) # trying to debug what's going wrong...
     if len(mod) == 0 :
         mod = datetime.now().astimezone().replace(microsecond=0).isoformat()
+    if date_only == "true":
+        date_only = '%Y-%m-%d'
+        mod = datetime.strptime(mod, '%Y-%m-%dT%H:%M:%S%z').strftime(date_only)
```
Comment (cicirello) [inline generatesitemap.py:228]: This is more complicated than it needs to be. At this point, before reformatting, `mod` is guaranteed to be formatted as a complete date and time. Just use a Python slice here to keep the date portion of that.

**Review by cicirello — CHANGES_REQUESTED** [review]

File: generatesitemap.py (hunk 1 above)
Comment (MarketingPip) [reply generatesitemap.py:228]: This is my problem I was struggling with. Can I get a clear example of the Boolean.. 

The action is set default to "false" which is a Boolean value. Tho when using a simple if statement: and nothing else.. things DO not work proper.... 

I would more than appreciate a demo with this. As I can not figure out what I am doing wrong.

File: generatesitemap.py [hunk 2]
```diff
@@ -345,7 +350,8 @@ def main(
         includePDF = sys.argv[4].lower() == "true",
         sitemapFormat = sys.argv[5],
         additionalExt = set(sys.argv[6].lower().replace(",", " ").replace(".", " ").split()),
-        dropExtension = sys.argv[7].lower() == "true"
+        dropExtension = sys.argv[7].lower() == "true",
+	date_only = sys.argv[8]
```
[the remark this answers is not in the document]
Comment (MarketingPip) [reply generatesitemap.py:354]: ... my bad - am I dumb. Is it literally just sys.arg[8] == True instead of "true" 🤦‍♀️

**Pull request closed without merging**